tunnel_map:
  footprint_width: 0.60
  footprint_length: 0.70
  lateral_clearance: 0.50
  obstacle_range: 0.5
  lookahead_distance: 1.5
  preferred_wall_side: 
  max_y_dist: 4.0
  preferred_wall_side: "LEFT"
  ransac_threshold: 0.1 
  publish_wall_dist: "true"
  pcl_topic: "/scan_cloud"
